Adding additional commands / improving embeds / app Avatar

Amoebawp opened the pull request:
added files : 
-- commands\servers\server-status.js
-- commands\servers\server-connect.js

modified files : 
-- commands\servers\server-info.js

Amoebawp committed:
Added status and connect cmds / beautified infos

Therosin commented:
function `GenPlayerList` doesnt exist in `server-status.js`,`server-connect.js`  this will cause an error,
need to fix those first, but those look good i think.

also are we still intending to show the full playerlist to everyone?
i mean we can realy.. though should probably modify `GenPlayerList` so as not to give steam-rep links and show steamId etc in those commands

Amoebawp committed:
Commit #2

Fixed GenPlayerList
Removed player list from info
Added server-players
Updated Avatar.png

Amoebawp commented:
Fixed missing `GenPlayerList` function
Removed player listing from `server-info`
Added `server-players` command
Updated Avatar.png

Amoebawp committed:
Commit 3

Fixed missing playersArray function

Amoebawp committed:
Commit 4

Fixing emotes into embed texts

Amoebawp committed:
Commit update

Fix missing emotes
Replaced Avatar.png

Amoebawp committed:
Update Avatar.png

Amoebawp committed:
Commit FINAL

Various minor adjustements for better clarity in embed texts

Amoebawp committed:
Commit FINAL #2

Fixed indentation for mobile devices
Slightly improved embed format
Finalized Avatar.png
